src/auth/AuthProvider.tsx

```diff
@@ -99,20 +108,7 @@ export const AuthProvider: React.FC = (props) => {
   };
 
   useEffect(() => {
-    check().then((resp) => {
```

Comment:
We're not, since (1) storing users locally has been removed, and (2) the initial auth check has moved to the backend. The only time this would run is if the user tries to load the dash, so a failing auth check would mean a broken app anyways.